nf_core/pipeline-template/lib/NfcoreTemplate.groovy

```diff
@@ -145,8 +151,10 @@ class NfcoreTemplate {
             } catch (all) {
                 // Catch failures and try with plaintext
                 def mail_cmd = [ 'mail', '-s', subject, '--content-type=text/html', email_address ]
-                if ( mqc_report.size() <= max_multiqc_email_size.toBytes() ) {
-                    mail_cmd += [ '-A', mqc_report ]
+                if (mqc_report) {
+                    if ( mqc_report.size() <= max_multiqc_email_size.toBytes() ) {
+                        mail_cmd += [ '-A', mqc_report ]
+                    }
```

Comment:
I think this can all be in one `if` statement..
```suggestion
                if (mqc_report && mqc_report.size() <= max_multiqc_email_size.toBytes() ) {
                    mail_cmd += [ '-A', mqc_report ]
```